els-front/src/plugins/localStorageHelper.js

```diff
@@ -13,3 +13,11 @@ export const getUserType = () =>{
 export const setUserType = (value) => {
     localStorage.setItem('access_type', window.btoa(value))
 }
+
+export const setPreviousPath = (value) => {
+    localStorage.setItem('previous_path',window.btoa(value))
+}
+
+export const getPreviousPath = () => {
+    return localStorage.getItem('previous_path') ? window.atob(localStorage.getItem('previous_path')) : ''
+}
```

Comment:
What is this used for?

Reply:
The purpose of this is to redirect to previous route when web page is refreshed sir but its not done yet

Reply:
On what instances? Maybe you can use navigate(-1) if applicable on the behavior you want